packages/flutter_tools/templates/create/lib/main.dart.tmpl

```diff
@@ -10,14 +10,14 @@ import 'package:{{pluginProjectName}}/{{pluginProjectName}}.dart';
 {{/withPluginHook}}
 
 {{^withDriverTest}}
-void main() => runApp(new MyApp());
+void main() => runApp(MyApp());
```

Comment:
I think you've picked up some extra changes, could you revert these?

Reply:
@jonahwilliams yeah I just noticed. I will look at all your suggestions and let you know as soon as I am done.